staging/publishing/rules.yaml

```diff
@@ -639,7 +641,10 @@ rules:
       dir: staging/src/k8s.io/kubectl
     name: release-1.15
     go: 1.12.5
+
     dependencies:
+      - repository: api
+        branch: master
```

Comment:
This should be `release-1.15` instead of `master`

Reply:
(related: created https://github.com/kubernetes/kubernetes/pull/79288 to fix rules for apimachinery and client-go)

Reply:
Thanks @nikhita for all of your suggestions and fixes! I will definitely implement everything when it's time to merge this in.

Reply:
@nikhita by the time it was time for me to merge these changes were already in place. Thanks again for reviewing